src/store/movie/action-creators.js

```diff
@@ -37,7 +36,7 @@ export const deleteMovie = (movieId) => {
             type: "DELETE_MOVIE_LIST_START"
         });
 
-        api.deleteMovie(`${movieId}`).then(() => {
+        api.deleteMovie(`${movieId}`).then((response) => {
```

Comment:
is there a need to add `${}` if we have nothing to concat? 🤔 
wouldn't `api.deleteMovie(movieId)` work?

Reply:
fixed